### ISSUE TITLE
A11y_SQL Azure Data Studio_Theme / Color Mode_ScreenReader: On tab key to the edit field  of the color theme list the placeholder is not read nor it is said that "enter the theme"

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#12985]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Oct2020;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.3.1;#Benchmark;#NVDA;#JAWS;#AILimited;#AINotInScope;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10
Screen reader: NVDA2020.3, JAWS2019.1912.1 ILM

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to button manage and press enter.
4. Use down arrow key and navigate to item color theme and press enter 

**Actual:**
When the list of themes are opened screen reader announces the list is opened and the list item that is selected, the placeholder  and enter to serach or apply a theme is not announced for the edit field that has current focus or when navigated to using tab.

**Issue observed with JAWS**

**Expected:**
When the focus is on the edit field the placeholder and text "please enter your theme" should be read followed by the list opened and the item.

**User Impact:**
The screen reader users who are low vision or color blind will not know that you can enter data to select a theme

**Attachment:**
[#12985_A11y_SQL Azure Data Studio_Theme -Color Mode_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/5401672/12985_A11y_SQL.Azure.Data.Studio_Theme.-Color.Mode_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
@fsteffi this is a feature we inherit from vscode, could you please open an issue here: https://github.com/microsoft/vscode

**Comment by fsteffi** [comment]
![image](https://user-images.githubusercontent.com/24580570/177981586-cdfac171-dd50-4150-be08-f9be0fb82660.png)
the issue is read the first item is annouced
Version: 1.37.0 (user setup)
Commit: d904740d93d7df76a0ba361f20e4351813b57645
Date: 2022-06-14T01:05:12.352Z
VS Code: 1.59.0
Electron: 13.6.6
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.22000
